pam_trimspaces.c

```diff
@@ -0,0 +1,41 @@
+#include <security/pam_modules.h>
+#include <stdio.h>
+#include <string.h>
+
+
+PAM_EXTERN int pam_sm_authenticate(pam_handle_t *pamh, int flags, int argc, const char **argv) {
+    const char *user;
+    int ret;
+
+    ret = pam_get_item(pamh, PAM_USER, (const void **) &user);
+    if (ret != PAM_SUCCESS)
+        return PAM_AUTH_ERR;

```

Comment:
In recent years I have never liked the idea of omitting braces for one-line statements. It seems like another `goto fail` waiting to happen.